DocGen4/Output/Module.lean

```diff
@@ -199,6 +199,9 @@ def moduleToHtml (module : Process.Module) : HtmlM Html := withTheReader SiteBas
   let relevantMembers := module.members.filter Process.ModuleMember.shouldRender
   let memberDocs ← relevantMembers.mapM (moduleMemberToHtml module.name)
   let memberNames := filterDocInfo relevantMembers |>.map DocInfo.getName
+  letI : MonadLift BaseHtmlM HtmlM := {
```

Comment:
Is this instance necessary? I would have hoped that Lean is able to infer this on its own.

Reply:
At lease the instances in https://github.com/acmepjz/doc-gen4/blob/bbf05929cc01df4150521802a61b76c11773fe80/DocGen4/Output/Base.lean#L113-L117 does not infer that automatically. What's worse, if I add `MonadLift BaseHtmlM HtmlM` as a global instance, there will be compile errors. So I only add this locally at this function.